pkg/kubernetes/pvc/v1alpha1/kubernetes.go

```diff
@@ -0,0 +1,97 @@
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// getClientSetFn is a typed function that abstracts
+//  fetching of clientset
+type getClientSetFn func() (clientset *kubernetes.Clientset, err error)
+
+// ListFn is a typed function that  abstracts listing
+// of persistent volume claims
+type ListFn func(cli *kubernetes.Clientset, namespace string, opts metav1.ListOptions) (*v1.PersistentVolumeClaimList, error)
+type kubeClientBuildOption func(*kubeclient)
```

Comment:
Explain kubeClientBuildOption in code comments

Reply:
added the comment